R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
@daroczig I'm sorry to hear about that re: CRAN.  As I understand it, CRAN also doesn't allow packages which install additional things without intervention by the end-user.  So, I think you're going to have to require the user to run a function to trigger the install of `AWR.jars` from drat.  

At which point, I think you're almost exactly duplicating what `awsjavasdk` already does (but adding a middle-person).  The only major difference I can think of is that, I don't load the `jar`s automatically (when they are present).  However, I'd gladly take a PR to modify .onLoad to make that behavior change.